content-types/article.yml

```diff
@@ -0,0 +1,14 @@
+name: Article
+description: A long form article
+id: article
+workflow: self-publish
+identifier: title
+attributes:
+  - type: text
```

Comment:
Shouldn't this have more than just a title attribute? It would be a short article without a block of text and only a title. Who knows? Maybe you're going for a tweeter thing? Who knows...